### ISSUE TITLE
use unsupervised GraphSAGE(SAGEConv) to classify nodes when only edges have features?

### ISSUE THREAD

**Issue by hex-16** [pyg-team/pytorch_geometric#3549]
Is there any way to use unsupervised GraphSAGE to classify nodes when only edges have features?
In more detail:
- nodes: without features
- edges: directed with features
- goal: node classification

When node A sampling A's neighbour node B, B's features are the features of directed edge A->B.
When node C sampling C's neighbour node B, B's features are the features of directed edge C->B.
Node A/C can sample node B only if directed edge A/C->B exits.
In other words, I have a class 'torch_geometric.data.data.Data' like:
Data(x=[num_nodes, target_embed_dim], edge_index=[2, num_edges], edge_attr=[num_edges, num_edge_features])
where x is a tensor of all zeros (means no features). 
Can I do this by reimplementing SAGEConv? If yes, how to reimplement it?

**Comment by rusty1s** [comment]
Closing this in favor of https://github.com/pyg-team/pytorch_geometric/discussions/3544
